src/asprof.cpp

```diff
@@ -53,3 +55,17 @@ DLLEXPORT asprof_error_t asprof_execute(const char* command, asprof_writer_t out
 DLLEXPORT asprof_thread_local_data* asprof_get_thread_local_data(void) {
     return ThreadLocalData::getThreadLocalData();
 }
+
+DLLEXPORT asprof_user_jfr_key asprof_create_user_jfr_key(const char* name) {
```

Comment:
As mentioned previously, I'm not in favor of having "user" in the names of public API functions. Whoever calls an API is already a user of this API.
What about `asprof_register_jfr_event`, `asprof_emit_jfr_event` or something like that?